aspnetcore/host-and-deploy/health-checks/samples/2.x/HealthChecksSample/ReadinessPublisher.cs

```diff
@@ -17,24 +16,18 @@ public ReadinessPublisher(ILogger<ReadinessPublisher> logger)
             _logger = logger;
         }
 
-        public List<(HealthReport report, CancellationToken cancellationToken)> 
-            Entries { get; } = 
-                new List<(HealthReport report, 
-                    CancellationToken cancellationToken)>();
-
-        public Exception Exception { get; set; }
-
         public Task PublishAsync(HealthReport report, 
             CancellationToken cancellationToken)
         {
-            Entries.Add((report, cancellationToken));
-
-            _logger.LogInformation("{TIMESTAMP} Readiness Probe Status: {RESULT}", 
-                DateTime.UtcNow, report.Status);
-
-            if (Exception != null)
+            if (report.Status == HealthStatus.Healthy)
+            {
+                _logger.LogInformation("{TIMESTAMP} Readiness Probe Status: {RESULT}", 
+                    DateTime.UtcNow, report.Status);
+            }
+            else
             {
-                throw Exception;
+                _logger.LogError("{TIMESTAMP} Readiness Probe Status: {RESULT}", 
```

Comment:
we don't normally show screaming caps for log value names - I don't want users to get the impression this is a convention we are recommending.

Reply:
We were ... perhaps in ancient times around here ... capitalizing placeholders in strings. It was asked/discussed back when it was started. I'll research it and report back later. You want this as `{timestamp}`?

Reply:
btw - this will be easy to fix everywhere. I've already made a note of it. I can RegEx the repo and change all of them on one PR. I'll open an issue and square that away at a later date.

Reply:
ok ... I **_didn't_** remember the convo correctly. It was Pascal case when talking to `anurse` about it ...

https://github.com/aspnet/AspNetCore.Docs/pull/4697#discussion_r148855863

I'll fix that up on the next commit. I'll address this everywhere in docs that it might exist on a new issue and another PR later.